style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+}
+
+/* header block */
+
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 375px;
+  height: 92px;
+  background-color: #3e3c3c;
+  position: absolute;
+  left: 0;
+  top: 0;
+  box-sizing: border-box;
+}
+
+#topHead,
+#bottomHead {
+  display: flex;
+  margin-left: 24px;
+}
+
+#time {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 16px;
+  letter-spacing: -0.28px;
+  width: 54px;
+  text-align: center;
+  margin-top: 16px;
+  margin-bottom: 24px;
+}
+
+#appBar {
+  display: flex;
+  margin-top: 10px;
+  margin-left: 170px;
+}
+
+.app {
+  margin: 5px;
+  list-style: none;
+}
+
+#logo {
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  margin-top: 0;
+  margin-bottom: 24px;
+}
+
+.unionImg {
+  display: flex;
+  align-items: center;
+  margin-left: 190px;
+  margin-top: 5px;
+}
+
+/* headline block */
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 375px;
+  height: 590px;
+  position: absolute;
+  left: 0;
+  top: 92px;
+  padding: 120px 24px;
+  box-sizing: border-box;
+  background-color: #1c1a19;
+}
+
+#title {
+  color: #ff6b00;
+  font-weight: 400;
+  font-size: 56px;
+  font-family: 'Crete Round', serif;
+  line-height: 64px;
+  flex-grow: 0;
+  margin: 12px 0;
+  letter-spacing: 0.37px;
+}
+
+#subtitle {
+  color: #fff5e1;
+  font-weight: 700;
+  font-size: 24px;
+  height: 27px;
+  flex-grow: 0;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  line-height: 32px;
+}
+
+#mainDescription {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  flex-grow: 0;
+  margin: 12px 0;
+  height: 123px;
+}
+
+#socialMedia {
+  list-style: none;
+  display: flex;
```

Comment:
- [ ] [REQUIRED] Your social media icons should be aligned to the left as seen in the template design. You may achieve this by adding CSS vaues of `padding: 0` to your `#socialMedia` selector.

| Your Portfolio | Template Design |
| --- | --- |
| ![Screenshot from 2022-04-20 18-30-07](https://user-images.githubusercontent.com/52934128/164289630-c4c424ec-91d6-4bd1-b362-ec63c947dd48.png) |![Screenshot from 2022-04-20 18-36-12](https://user-images.githubusercontent.com/52934128/164289854-837695ae-89a7-41a8-be14-ce505dad65a5.png) |